addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -358,7 +358,7 @@ def action_get_attachment_tree_view(self):
     def action_applications_email(self):
         return {
             'type': 'ir.actions.act_window',
-            'name': _('Applications'),
+            'name': _('Candidacies'),
```

Comment:
Not sure 'Candidacies' applies in this context.
```suggestion
            'name': _('Job Applications'),
```